specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,3090 @@
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Submit Analysis Job Request": {
+            "$ref": "./examples/SuccessfulAnalyzeTextJobsMultipleTaskSubmitRequest.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/analyze-text/jobs/{jobId}": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "description": "Get the status of an analysis job.  A job may consist of one or more tasks.  Once all tasks are completed, the job will transition to the completed state and results will be available for each task.",
+        "operationId": "AnalyzeText_JobStatus",
+        "summary": "Get analysis status and results",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/JobId"
+          },
+          {
+            "$ref": "#/parameters/ShowStats"
+          },
+          {
+            "default": 20,
+            "description": "(Optional) Set the maximum number of results per task. When both top and skip are specified, skip is applied first.",
+            "in": "query",
+            "maximum": 50,
+            "minimum": 1,
+            "name": "top",
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "default": 0,
+            "description": "(Optional) Set the number of elements to offset in the response. When both top and skip are specified, skip is applied first.",
+            "in": "query",
+            "minimum": 0,
+            "name": "skip",
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Analysis job status and metadata.",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeTextJobState"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Text Analysis Job Status Request": {
+            "$ref": "./examples/SuccessfulAnalyzeTextJobsMultipleTaskStatusRequest.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AnalyzeTextTaskKind": {
+      "type": "string",
+      "description": "Enumeration of supported Text Analysis tasks.",
+      "enum": [
+        "SentimentAnalysis",
+        "EntityRecognition",
+        "PiiEntityRecognition",
+        "KeyPhraseExtraction",
+        "LanguageDetection",
+        "EntityLinking",
+        "Healthcare",
```

Comment:
NIt: Is `Healthcare` is a too general name?

Reply:
I remember having a really long discussion about the naming during the original design of the Healthcare SDK.  I think we landed on `Healthcare` in order to remain somewhat generic to prevent any assumptions made about what the API's capabilities are, but if you have a suggestion for other names maybe we can discuss offline.

We will be including other terminology changes in the next preview, so if this is something that you feel strongly about we can include it with other terminology changes as well.